### PR TITLE
Committee persistent storage

### PR DESCRIPTION
**Pull request by OmriEshhar1**
This change is [Reviewable](https://reviewable.io/reviews/starkware-libs/starkex-data-availability-committee/2)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,13 @@ Typically the reference batch is the immediate predecessor of the new batch. How
 nature of the blockchain, it is possible that a batch created by the Starkware application is later
 reverted and replaced by a different batch.
 
+## Setting-up Persistent Data Storage
+Before you start, you will need to set up a storage directory for the committee Aerospike database.
+To do that:
+- Create a directory. Disk-space requirement: 40 Gb.
```
Comment (gkaempfer) [inline README.md]: Gb -> GB

**Review by gkaempfer — CHANGES_REQUESTED** [review]
> Reviewed 2 of 3 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/starkex-data-availability-committee/2)* status: all files reviewed, 2 unresolved discussions (waiting on @OmriEshhar1)

___
*[`README.md` line 96 at r2](https://reviewable.io/reviews/starkware-libs/starkex-data-availability-committee/2#-NGGLiJD-b2lHJScTMng:-NGGLiJD-b2lHJScTMnh:b-bpydrd) ([raw file](https://github.com/starkware-libs/starkex-data-availability-committee/blob/e323d93d8e7fd97190ff4e5053c4499791122a34/README.md#L96)):*
> ```Markdown
> for the production environment:
> 
> - EC2 is recommended.
> ```

Runner the docker images directly on separate EC2 instances is recommended.
___
*[`README.md` line 97 at r2](https://reviewable.io/reviews/starkware-libs/starkex-data-availability-committee/2#-NGGLNt-DxPi1efJOORO:-NGGLNt0B3kHmFMn8XPi:bvqkff3) ([raw file](https://github.com/starkware-libs/starkex-data-availability-committee/blob/e323d93d8e7fd97190ff4e5053c4499791122a34/README.md#L97)):*
> ```Markdown
> 
> - EC2 is recommended.
>   - Alternatively, multiple instances of docker-compose should be deployed to offer some resilience.
> ```

move to mandatory signers section. change the text so it implies that if you don't run with RF 2 you should run multiple nodes with RF 1. This is requires so that in the case of data loss by one of the non-resilient DBs other nodes will continue to function as DAC members without introducing unnecessary latency.

**Review by gkaempfer — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/starkex-data-availability-committee/2)* status: all files reviewed, 3 unresolved discussions (waiting on @OmriEshhar1)

___
*[`README.md` line 89 at r2](https://reviewable.io/reviews/starkware-libs/starkex-data-availability-committee/2#-NGGOi2H-ivQSOnKmfCX:-NGGOi2H-ivQSOnKmfCY:b7y06m8) ([raw file](https://github.com/starkware-libs/starkex-data-availability-committee/blob/e323d93d8e7fd97190ff4e5053c4499791122a34/README.md#L89)):*
> ```Markdown
>   - For Aerospike this will require the enterprise edition. For more details [click here](https://docs.aerospike.com/server/operations/configure/consistency).
> - Recommend setting a `replication-factor` of `2` (the default is `1`).
>   - This will require setting up 3 servers.
> ```

3 DB servers

**Review by OmriEshhar1 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/starkex-data-availability-committee/2)* status: all files reviewed, 3 unresolved discussions (waiting on @gkaempfer)

___
*[`README.md` line 89 at r2](https://reviewable.io/reviews/starkware-libs/starkex-data-availability-committee/2#-NGGOi2H-ivQSOnKmfCX:-NGGQQuP2ERFr5ZjbGnv:b-896fix) ([raw file](https://github.com/starkware-libs/starkex-data-availability-committee/blob/e323d93d8e7fd97190ff4e5053c4499791122a34/README.md#L89)):*
<details><summary><i>Previously, gkaempfer wrote…</i></summary><blockquote>

3 DB servers
</blockquote></details>

Done.
___
*[`README.md` line 96 at r2](https://reviewable.io/reviews/starkware-libs/starkex-data-availability-committee/2#-NGGLiJD-b2lHJScTMng:-NGGQRAJ3BPkov0s2wjZ:b-896fix) ([raw file](https://github.com/starkware-libs/starkex-data-availability-committee/blob/e323d93d8e7fd97190ff4e5053c4499791122a34/README.md#L96)):*
<details><summary><i>Previously, gkaempfer wrote…</i></summary><blockquote>

Runner the docker images directly on separate EC2 instances is recommended.
</blockquote></details>

Done.
___
*[`README.md` line 97 at r2](https://reviewable.io/reviews/starkware-libs/starkex-data-availability-committee/2#-NGGLNt-DxPi1efJOORO:-NGGQRX80YGqvpUG4Em2:b-896fix) ([raw file](https://github.com/starkware-libs/starkex-data-availability-committee/blob/e323d93d8e7fd97190ff4e5053c4499791122a34/README.md#L97)):*
<details><summary><i>Previously, gkaempfer wrote…</i></summary><blockquote>

move to mandatory signers section. change the text so it implies that if you don't run with RF 2 you should run multiple nodes with RF 1. This is requires so that in the case of data loss by one of the non-resilient DBs other nodes will continue to function as DAC members without introducing unnecessary latency.
</blockquote></details>

Done.

**Review by gkaempfer — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/starkex-data-availability-committee/2)* status: 2 of 3 files reviewed, 2 unresolved discussions (waiting on @OmriEshhar1)

___
*[`README.md` line 99 at r3](https://reviewable.io/reviews/starkware-libs/starkex-data-availability-committee/2#-NGGROAA6muT-s9TK3Yy:-NGGROAA6muT-s9TK3Yz:b-oxdsj) ([raw file](https://github.com/starkware-libs/starkex-data-availability-committee/blob/f8daa223f105e9aa4a6f1730f5eac272559283a0/README.md#L99)):*
> ```Markdown
> - It is recommended to run the docker images on separate EC2 instances.
> 
> - Alternatively, deploy multiple instances of docker-compose.
> ```

this alternative isn't required IMHO

**Review by gkaempfer — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/starkex-data-availability-committee/2)* status: 2 of 3 files reviewed, 2 unresolved discussions (waiting on @OmriEshhar1)

___
*[`README.md` line 91 at r3](https://reviewable.io/reviews/starkware-libs/starkex-data-availability-committee/2#-NGGRYnzBbaC3X_-lp4M:-NGGRYnzBbaC3X_-lp4N:bctuf01) ([raw file](https://github.com/starkware-libs/starkex-data-availability-committee/blob/f8daa223f105e9aa4a6f1730f5eac272559283a0/README.md#L91)):*
> ```Markdown
>   - This will require setting up 3 DB servers.
> 
> - Alternatively, run multiple instances of the committee on separate EC2 instances (without strong consistency).
> ```

and with RF 1

**Review by OmriEshhar1 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/starkex-data-availability-committee/2)* status: 2 of 3 files reviewed, 2 unresolved discussions (waiting on @gkaempfer)

___
*[`README.md` line 91 at r3](https://reviewable.io/reviews/starkware-libs/starkex-data-availability-committee/2#-NGGRYnzBbaC3X_-lp4M:-NGGWt_QARqxM4-vNRjS:b-896fix) ([raw file](https://github.com/starkware-libs/starkex-data-availability-committee/blob/f8daa223f105e9aa4a6f1730f5eac272559283a0/README.md#L91)):*
<details><summary><i>Previously, gkaempfer wrote…</i></summary><blockquote>

and with RF 1
</blockquote></details>

Done.
___
*[`README.md` line 99 at r3](https://reviewable.io/reviews/starkware-libs/starkex-data-availability-committee/2#-NGGROAA6muT-s9TK3Yy:-NGGWtKwDai_aVc6wdC9:b-896fix) ([raw file](https://github.com/starkware-libs/starkex-data-availability-committee/blob/f8daa223f105e9aa4a6f1730f5eac272559283a0/README.md#L99)):*
<details><summary><i>Previously, gkaempfer wrote…</i></summary><blockquote>

this alternative isn't required IMHO
</blockquote></details>

Done.

**Review by gkaempfer — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/starkex-data-availability-committee/2#-:-NGGe3VA3mQpLSfpTm8z:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @OmriEshhar1)

**Pull request merged**